addons/website_blog/static/tests/tours/blog_tags_tour.js

```diff
@@ -0,0 +1,60 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+
+/**
+ * Makes sure that blog tags can be created and removed.
+ */
+tour.register('blog_tags', {
+    test: true,
+    url: '/blog',
+}, [{
+        trigger: "article[name=blog_post] a",
+        content: "Go to first blog.",
+    }, {
+        trigger: "section#o_wblog_post_main",
+        content: "Wait for single blog post.",
+        run: () => {}, // it's a check
```

Comment:
I am changing to:
- Using extra_trigger for this specific second step (I think it's nice to make a dedicated step for steps to "verify / it's a check" but when it's not on a step we actually really want to "verify", I'd say extra_trigger is better to speed things up :+1: 
- Usually I don't think we put a point at the end of "content" for test tours. I'll entertain my OCD and remove them :wink: 
- A while ago, RDE asked me to change to have `content` before `trigger`, I'll do it here too, I agree this is better :)